cms/static/sass/elements/_controls.scss

```diff
@@ -336,6 +336,14 @@
     &.toggle-action {
       // TODO: generalize and move checkbox styling in from static-pages and assets sass
     }
+
+    .btn-default.delete-button {
```

Comment:
I don't think this needs ``.btn-default`` here. Are there situations where delete buttons do want the border? I'm still somewhat confused what the class ``btn-default`` means.